Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/FocusHandlerTrigger.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Utility component to hook up Unity events to the OnFocusEnter and OnFocusExit events.
+    /// </summary>
+    public class FocusHandlerTrigger : MonoBehaviour, IMixedRealityFocusHandler
+    {
+        /// <summary>
```

Comment:
Do these need to be initialized? Also, I believe MRTK aims to use the pattern internal private serialized fields, public getters and setters.

Reply:
They shouldn't need to be initialized since Unity's we do that for you for any serializable properties. But, I'm happy to provide a default. Also, I do prefer the private serialized field pattern. I was just following what I found in PressableButton, BoundingBox, etc. But, we should probably fix those too? I'll make this change!